lbz/api_base.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
+
+
+class APIBase:
```

Comment:
I think we need to discuss the naming that is used here as I understand this is a base class but both Resource and Broker inherit from it and that is a little bit strange for me, at least at the first glance 🤔